connector/src/main/java/org/jboss/as/connector/subsystems/jca/BootstrapContextAdd.java

```diff
@@ -59,13 +59,16 @@ protected void performRuntime(final OperationContext context, final ModelNode op
         String name = JcaBootstrapContextDefinition.BootstrapCtxParameters.NAME.getAttribute().resolveModelAttribute(context, model).asString();
         String workmanager = JcaBootstrapContextDefinition.BootstrapCtxParameters.WORKMANAGER.getAttribute().resolveModelAttribute(context, model).asString();
         boolean usingDefaultWm = false;
+        CloneableBootstrapContext ctx;
         if (DEFAULT_NAME.equals( workmanager)) {
             usingDefaultWm = true;
+            ctx = new NamedBootstrapContext(name);
+        } else {
+            ctx = new NamedBootstrapContext(name,workmanager);
```

Comment:
Why are these changes required? The referenced JIRA's only mention issues with tests yet both this and the change above don't seem to be in the tests.